pkg/config/config.go

```diff
@@ -139,6 +139,10 @@ func FindRegalDirectory(path string) (*os.File, error) {
 			return nil, errors.New("stopping as dir is empty string")
 		}
 
+		if len(parts) < 2 {
+			return nil, errors.New("stopping as dir is root directory")
+		}
```

Comment:
Is "root directory" well defined for windows? How about `C:\` or `D:\` or `\\`?

Reply:
Yeah, no idea. I hope this code isn’t permanent, as I’ll want to find out what’s going on here… but first to get a windows box somehow.